plugins/realtek-mst/README.md

```diff
@@ -31,19 +31,13 @@ This plugin supports the following protocol ID:
 
 ## GUID Generation
 
-Devices use an extra instance ID derived from SMBIOS, e.g.
+These devices use the standard I2C DeviceInstanceId values, e.g.
 
-* `I2C\NAME_10EC2142:00&FAMILY_Google_Hatch`
+* `I2C\NAME_1AF80175:00`
```

Comment:
Is this going to break anything that uses the old way?

Reply:
There's no LVFS firmware for this hardware at all, I'm really trying to stop a different vendor from copying this semi-broken design.

Reply:
Ah okay good.